### PR TITLE
olya_task8_finding_the_longest_word_in_the_file

### PR DESCRIPTION
**Pull request by olyapetryshyn**

### CODE REVIEW

File: olya_petryshyn/finding_the_longest_word.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Finding the longest word in the file.
+"""
+import string
+file_name = 'textfile.txt'
+
+def longest_word(file_name):
+    with open(file_name, 'r') as file:
+        string_text = file.read().replace('\n', ' ')
+    list_text = string_text.split()
+    list_text = [''.join(character for character in word if character not in string.punctuation) for word in list_text]
```
Comment (pivancy) [inline olya_petryshyn/finding_the_longest_word.py]: What the magic are you doing here?

Comment (olyapetryshyn) [reply olya_petryshyn/finding_the_longest_word.py]: Here I'm removing punctuation signs from text from file to find the longest word. Does it look that bad?)

Comment (pivancy) [reply olya_petryshyn/finding_the_longest_word.py]: This like looks super strange.

Comment (olyapetryshyn) [reply olya_petryshyn/finding_the_longest_word.py]: Fixed.

File: olya_petryshyn/task0008_finding_the_longest_word.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+"""
+Finding the longest word in the file.
+"""
+import string
+file_name = 'textfile.txt'
+
+def the_longest_word(file_name):
+    with open(file_name, 'r') as file:
+        string_text = file.read().replace('\n', ' ')
+    for item in string.punctuation:
```
Comment (pivancy) [inline olya_petryshyn/task0008_finding_the_longest_word.py]: This part might be replaced with:

    string_text = string_text.translate(None, string, punctuation)

File: olya_petryshyn/task0008_finding_the_longest_word.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        string_text = string_text.replace(item, '')
+    list_text = string_text.split()
+    list_text.sort(key=lambda x: len(x))
```
Comment (pivancy) [inline olya_petryshyn/task0008_finding_the_longest_word.py]: `max(list_text, key=len)` should works fine for that!

File: olya_petryshyn/task0008_finding_the_longest_word.py (hunk 2 above)
Comment (olyapetryshyn) [reply olya_petryshyn/task0008_finding_the_longest_word.py]: Done.

File: olya_petryshyn/task0008_finding_the_longest_word.py (hunk 3 above)
Comment (olyapetryshyn) [reply olya_petryshyn/task0008_finding_the_longest_word.py]: Done.

**Comment by pivancy** [comment]
👍

**Pull request merged**